modules/runner/build.gradle

```diff
@@ -158,8 +158,21 @@ dependencies {
     integrationTestImplementation libs.typesafe.config
     integrationTestImplementation libs.auto.service.annotations
 
+    testFixturesImplementation project(':ignite-core')
     testFixturesImplementation project(':ignite-api')
+    testFixturesImplementation project(':ignite-raft-api')
+    testFixturesImplementation project(':ignite-raft')
     testFixturesImplementation project(':ignite-sql-engine')
+    testFixturesImplementation project(':ignite-table')
+    testFixturesImplementation project(':ignite-schema')
+    testFixturesImplementation testFixtures(project(':ignite-api'))
+    testFixturesImplementation testFixtures(project(':ignite-core'))
+    testFixturesImplementation testFixtures(project(':ignite-network'))
+    testFixturesImplementation libs.jetbrains.annotations
+    testFixturesImplementation libs.junit5.api
+    testFixturesImplementation libs.junit5.impl
+    testFixturesImplementation libs.junit5.params
```

Comment:
Do we really need all of this?

Look like these dependencies already added here
`apply from: "$rootDir/buildscripts/java-test-fixtures.gradle"`

Reply:
Yes, you are right. Fixed.